### DISCUSSION TITLE
False Positives

### DISCUSSION THREAD

**Discussion by fldave850** [gophish/gophish#2833] [Ideas]
Has there been any research into eliminating/filtering false positives by using a list of known email security tool IP's, proxies, bots, tor, blacklisted IP's, etc? 

- Maintaining a repository of known email security IP's like Microsoft, Google, Proofpoint, Mimecast, SpamTitan, etc.
- Using a tool like IPStack's Security Module - https://ipstack.com/documentation - to determine proxy, tor, etc.
- Using IPsum to filter bad IP's - https://github.com/stamparm/ipsum
- Flagging events that are from multiple IP's, agents, locations, etc as possible false positives for a single target.

This could reference Issues:

https://github.com/gophish/gophish/issues/2803
https://github.com/gophish/gophish/issues/2679
https://github.com/gophish/gophish/issues/2357

**Comment by ConsoleScript** [comment]
I created an NGINX proxy. I send them to the captcha page, if they can't pass it, they don't fetch ?rid and it won't be marked as clicked. NGINX + JS page (one static only). Rest is up to your imagination.

**Comment by mary8588** [comment]
dpatil@gpv.com
